### PR TITLE
Add Travis and Coveralls, to provide complex build matrix

### PR DESCRIPTION
**Pull request by gwax**
Travis allows us to build against multiple versions, which makes it a good choice for open source releases with complex testing needs..

### CODE REVIEW

**Comment by gwax** [comment]
This PR depends on #10

File: setup.py [hunk 1]
```diff
@@ -1,34 +1,54 @@
...
-exec(open('temporal-sqlalchemy/version.py', 'r').read())  # pylint: disable=exec-used
+exec(open('temporal_sqlalchemy/version.py', 'r').read())
 
+SHORT_DESCRIPTION = 'Temporal Extensions for SQLAlchemy ORM'
+LONG_DESCRIPTION = open('README.rst', 'r').read()
 
-setup(
+DEPENDENCIES = [l.strip() for l in open('requirements.txt', 'r')]
+if sys.version_info < (3, 5):
+    DEPENDENCIES.append('typing>=3.5.2,<4.0.0')
```
Comment (multigl) [inline setup.py]: does this also include a backfill for `typing.Type`?

Comment (gwax) [inline setup.py]: Annoyingly, it provides `typing.Type` for Python < 3.5 but not for 3.5.0<=Python<3.5.2

Comment (multigl) [inline setup.py]: WEAK oh well

**Comment by dargueta-work** [comment]
@gwax I have a PR for making this compatible with Python 2.7+; it required removal of _all_ type hinting the 3.5 way and moving it into comments: https://github.com/CloverHealth/temporal-sqlalchemy/pull/15

**Pull request merged**